mon-pix/translations/nl.json

```
@@ -1575,7 +1575,7 @@
         "e-learning": "Afstandsonderwijs",
         "hybrid-training": "Hybride training",
         "in-person-training": "Persoonlijke training",
-        "modulix": "Modulix",
+        "modulix": "Module Pix",
```

Il semberait que en nerlandais c'est :

```suggestion
        "modulix": "Pix Module",
```